### PR TITLE
fix(liquidator, disputer): Exit early if EMP is expired

### LINKED ISSUE #1949: liquidator bot tries to liquidate position in expired contract
**Description**

Pagerduty went off trying to liquidate position in expired contract yusd-sept. Gives out of gas error probably due to revert.
```js
[error] usdeth-mainnet-liquidator (Liquidator)⭢Failed to liquidate position: not enough synthetic (or large enough approval) to initiate liquidation:x:
• sponsor: 0xDc0...09ad5
• inputPrice: 2578201047065322
• position:
   - sponsor: 0xDc0...09ad5
- withdrawalRequestPassTimestamp: 1598894602
- withdrawalRequestAmount: 47000000000000000000
- numTokens: 49629108321645473000000
- amountCollateral: 206248053597079775598
- hasPendingWithdrawal: true
• minLiquidationPrice: 0
• maxLiquidationPrice: 3222751308831652
• tokensToLiquidate: 49629108321645473000000
• error: Error: gas required exceeds allowance (12493867) or always failing transaction
   at WebsocketSubprovider._handleSocketMessage (/protocol/node_modules/@trufflesuite/web3-provider-engine/subproviders/websocket.js:121:18)
   at WebSocket.onMessage (/protocol/node_modules/ws/lib/event-target.js:120:16)
   at WebSocket.emit (events.js:315:20)
   at Receiver.receiverOnMessage (/protocol/node_modules/ws/lib/websocket.js:720:20)
   at Receiver.emit (events.js:315:20)
   at Receiver.dataMessage (/protocol/node_modules/ws/lib/receiver.js:414:14)
   at /protocol/node_modules/ws/lib/receiver.js:371:23
   at /protocol/node_modules/ws/lib/permessage-deflate.js:307:9
   at /protocol/node_modules/ws/lib/permessage-deflate.js:389:7
   at afterWrite (_stream_writable.js:490:5)
   at onwrite (_stream_writable.js:470:7)
   at InflateRaw.afterTransform (_stream_transform.js:98:3)
   at Zlib.processCallback (zlib.js:578:8)
```
https://etherscan.io/tx/0x997c0f15d4c778da0bd18eaf10c3b5ebe7dd980bcfd8e2b0f9506038320be1d4

**Reproduction**

Run tests on liquidator against position in an expired contract. It will probably try to liquidate it. 

**Expected Behavior**

Bot should ignore expired contracts.

**Screenshots**

image

**Version and OS Info**

 - OS: [e.g. Ubuntu]
 - Version [either git hash of the repository or release]

**Additional Context**

### PR DESCRIPTION
**Pull request by nicholaspai**
Signed-off-by: Nick Pai <npai.nyc@gmail.com>

**Motivation**

#1949

**Summary**

Log and exit early without throwing if EMP is expired

**Issue(s)**

Fixes #1949

### CODE REVIEW

File: packages/common/src/FormattingUtils.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const networkUtils = require("./PublicNetworks");
 
 const BigNumber = require("bignumber.js");
 const moment = require("moment");
+const assert = require("assert");
```
Comment (nicholaspai) [inline packages/common/src/FormattingUtils.js:5]: This was causing my local test to throw, not sure why it wasn't caught in CI?

Comment (nicholaspai) [reply packages/common/src/FormattingUtils.js:5]: @daywiss FYI

File: packages/liquidator/index.js [hunk 2]
```diff
@@ -205,7 +220,8 @@ async function run({
       }
       logger.debug({
         at: "Liquidator#index",
-        message: "End of execution loop - waiting polling delay"
+        message: "End of execution loop - waiting polling delay",
+        pollingDelay: `${pollingDelay} (s)`
```
Comment (nicholaspai) [inline packages/liquidator/index.js:224]: Unrelated but thought this would be useful

File: packages/common/src/FormattingUtils.js (hunk 1 above)
Comment (daywiss) [reply packages/common/src/FormattingUtils.js:5]: thats really wierd. might not be able to build on front end, its a node module

Comment (daywiss) [reply packages/common/src/FormattingUtils.js:5]: oh im not sure why it wouldnt run locally though... its like a standard thing. unless it was actually asserting some thing to be true

Comment (nicholaspai) [reply packages/common/src/FormattingUtils.js:5]: yeah you just have to import the assert module though

**Review by daywiss — APPROVED** [review]

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**